package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "opentelemetry-sandbox-web-js",
+    "version": "0.1.0",
+    "description": "OpenTelemetry non-production level experimental Web JS packages.",
+    "keywords": [
+        "opentelemetry",
+        "browser",
+        "profiling",
+        "metrics",
+        "stats"
+      ],
+      "author": "OpenTelemetry Authors",
+      "license": "Apache-2.0",
+      "scripts": {
+        "do-sandbox-repo-merge": "cd sandbox-tools/merge-repos && npm install && cd ../.. && npm run do-repo-merge --prefix sandbox-tools/merge-repos"
```

Comment:
might be worth checking in package-lock and doing `npm ci` in this instance since the merge-tools isn't meant to be published and consumed.

Reply:
This is going to be merged into `main` so the lock could get confusing once we start making it compile.
We can always add later and people (unless working on the script) should not really be running this directly

Reply:
Oh, I see your point now *doh!* I set it up with the script folder needing it's own `npm install` -- will add.